go-controller/pkg/ovn/gateway.go

```diff
@@ -17,6 +17,35 @@ func (ovn *Controller) getOvnGateways() ([]string, error) {
 	return strings.Fields(string(out)), err
 }
 
+func (ovn *Controller) getGatewayPhysicalIP(physicalGateway string) string {
```

Comment:
Any reason to not just return an error from this function if the IP is ""?  Looks like most users have to C&P the error message code, might as well do it in one place.

Reply:
Makes sense. I decided to not print any errors in the function itself and leave that responsibility to caller.